views/index.ejs

```diff
@@ -22,22 +22,43 @@
 
     <body>
         <div id="overlay-backdrop"></div>
-        <%- include("top-nav-view", {user}) %>
-        <div id="contents">
+        <div id="contents" class="contents">
             <div id="notification-overlay-container"></div>
             <div id="overlay-container"></div>
-            <div id="upload-center">
-                <div id="upload-area">
-                    <div id="upload-preview"></div>
-                    <div class="button" id="select-button">
-                        <form action="/upload" id="upload" method="POST" enctype="multipart/form-data">
-                            <input type="file" accept=".gif, .png, .jpg, .jpeg, .bmp" id="select-me" value="" />
-                        </form>
-                        <span class="collecticons collecticons-picture head-icon"></span>
-                    </div>
-                    <div class="button" id="upload-button">
-                        <span class="collecticons collecticons-upload-2 head-icon"></span>
+            <div class="home-wrapper">
+                <%- include("top-nav-view", {user}) %>
+                <div class="home-content-wrapper">
+                    <div class="space"></div>
+                    <img class="logo" src="/assets/images/logo.svg" />
+                    <div id="upload-center" class="upload-center">
+                        <div id="upload-area" class="upload-area">
+                            <div id="upload-preview" class="upload-preview"></div>
+                            <div class="button select" id="select-button">
+                                <form action="/upload" id="upload" method="POST" enctype="multipart/form-data">
+                                    <input type="file" accept=".gif, .png, .jpg, .jpeg, .bmp" id="select-me" value="" />
+                                </form>
+                                <span class="collecticons collecticons-picture head-icon"></span>
+                            </div>
+                            <div class="button upload" id="upload-button">
+                                <span class="collecticons collecticons-upload-2 head-icon"></span>
+                            </div>
+                            <div class="info-wrapper">
+                                <div class="info-bubble hidden" id="upload-info">
+                                    <ul>
+                                        <li>Filetypes accepted: <%= validImageTypesString %></li>
+                                        <% if (expireTimeString != null) { %>
```

Comment:
I should either update this to check for any expire time strings not `"unspecified"`, or I should make the return value for `getExpireTimeString` when invalid parameters are passed be `null`.